laws/src/main/scala/cats/laws/ArrowChoiceLaws.scala

```diff
@@ -0,0 +1,43 @@
+package cats
+package laws
+
+import cats.arrow.ArrowChoice
+import cats.syntax.compose._
+
+/**
+ * Laws that must be obeyed by any `cats.arrow.ArrowChoice`.
+ */
+trait ArrowChoiceLaws[F[_, _]] extends ArrowLaws[F] with ChoiceLaws[F] {
+  implicit override def F: ArrowChoice[F]
+  implicit def Function: ArrowChoice[Function1]
```

Comment:
Are these laws based on the ones defined in [Control-ArrowChoice](https://hackage.haskell.org/package/base-4.10.1.0/docs/Control-Arrow.html)?

Reply:
Hmm, they are meant to but interestingly the ones in the source and the ones in the Haskell doc do not quite look the same, I'll give them another pass tonight to make sure I transcribed the correct set.

Reply:
Yeah, I totally misread them, somehow. How embarrassing! Sorry about that. Fixing.